graylog2-server/src/main/java/org/graylog2/database/MongoConnection.java

```diff
@@ -18,10 +18,12 @@
 
 import com.mongodb.DB;
 import com.mongodb.Mongo;
-import com.mongodb.MongoClient;
+import com.mongodb.client.MongoDatabase;
 
 public interface MongoConnection {
     Mongo connect();
 
     DB getDatabase();
```

Comment:
I think we should mark this as `@Deprecated` so we can remove it for the next major release.